tests/storages_tests/journal_tests/test_journal.py

```diff
@@ -28,33 +28,43 @@
 from optuna.testing.tempfile_pool import NamedTemporaryFilePool
 
 
-LOG_STORAGE = [
-    "file_with_open_lock",
-    "file_with_link_lock",
-    "redis_default",
-    "redis_with_use_cluster",
+LOG_STORAGE_WITH_PARAMETER = [
+    ("file_with_open_lock", 3),
+    ("file_with_open_lock", 30),
+    ("file_with_open_lock", None),
+    ("file_with_link_lock", 3),
+    ("file_with_link_lock", 30),
+    ("file_with_link_lock", None),
```

Comment:
I think `grace_period=3` is redundant.

```suggestion
    ("file_with_open_lock", 30),
    ("file_with_open_lock", None),
    ("file_with_link_lock", 30),
    ("file_with_link_lock", None),
```